catalog/solr/pom.xml

```diff
@@ -30,6 +30,11 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.7</version>
             </dependency>
+            <dependency>
+                <groupId>commons-fileupload</groupId>
+                <artifactId>commons-fileupload</artifactId>
+                <version>1.3.2</version>
```

Comment:
I'm okay with it since it actually works as is, I'm just a fan of using properties as much as possible to prevent future maintenance issues.